### PR TITLE
[13.0][FIX] sale_stock_available_info_popup: immediately_usable_qty_today triggers

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
Add missing triggers of computational field immediately_usable_qty_today

 TT38197

### CODE REVIEW

File: sale_stock_available_info_popup/models/sale_order.py [hunk 1]
```diff
@@ -10,13 +10,21 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     immediately_usable_qty_today = fields.Float(
-        compute="_compute_immediately_usable_qty_today"
+        compute="_compute_immediately_usable_qty_today", store=True
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py]: This can't be stored, as it depends on context.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And please follow commit message guidelines including module name.

File: sale_stock_available_info_popup/models/sale_order.py [hunk 2]
```diff
@@ -13,10 +13,19 @@ class SaleOrderLine(models.Model):
         compute="_compute_immediately_usable_qty_today"
     )
 
-    @api.depends("product_id", "product_uom_qty")
+    @api.depends(
+        "product_id",
+        "product_uom_qty",
+        "product_id.immediately_usable_qty",
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py]: This shouldn't be added, as only direct things that you change in your screen will provoke the recomputation, and you can't change this data through it.

File: sale_stock_available_info_popup/models/sale_order.py [hunk 3]
```diff
@@ -13,10 +13,19 @@ class SaleOrderLine(models.Model):
...
+        "order_id.commitment_date",
+        "order_id.date_order",
+        "order_id.state",
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py]: Why adding this one?

File: sale_stock_available_info_popup/models/sale_order.py [hunk 4]
```diff
@@ -13,10 +13,19 @@ class SaleOrderLine(models.Model):
...
+        "display_qty_widget",
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py]: And this one?

File: sale_stock_available_info_popup/models/sale_order.py [hunk 5]
```diff
@@ -13,10 +13,19 @@ class SaleOrderLine(models.Model):
...
+        "warehouse_id",
+    )
     def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
-        self.immediately_usable_qty_today = False
+        self.immediately_usable_qty_today = 0
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/models/sale_order.py]: Is there any difference?

File: sale_stock_available_info_popup/tests/test_sale_stock_available_info_popup.py [hunk 6]
```diff
@@ -49,7 +58,7 @@ def _add_sale_line(self, sale_form, product, qty=1):
             line.product_id = self.product
             line.product_uom_qty = qty
 
-    @users("admin", "demo")
+    @users("admin", "stock_user")
```
Comment (pedrobaeza) [inline sale_stock_available_info_popup/tests/test_sale_stock_available_info_popup.py]: Why this change?

**Comment by chienandalu** [comment]
Changes applied

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Is this needed in v14?

**Comment by chienandalu** [comment]
Well, the module isn't migrated so... :sweat_smile:

**Pull request merged**